mmv1/products/identityplatform/ProjectDefaultConfig.yaml

```diff
@@ -21,9 +21,12 @@ description: |
   There is no persistent data associated with this resource.
 create_verb: :PATCH
 delete_verb: :PATCH
+deprecation_message: >-
+  Deprecated. Use the `google_identity_platform_config` resource instead.
+  It contains a more comprehensive list of fields, and was created before
+  google_identity_platform_project_default_config` was added.
```

Comment:
As long as we're making this change it might make sense to update the deprecation message to match the format from https://googlecloudplatform.github.io/magic-modules/develop/make-a-breaking-change/#resource-deprecation-due-to-removal-or-rename

```suggestion
  `google_identity_platform_config` is deprecated and will be removed in the next major release
  of the provider. Use the `google_identity_platform_config` resource instead. It contains a more
  comprehensive list of fields, and was created before `google_identity_platform_project_default_config`
  was added.
```